### PR TITLE
[composite compliance] istft

### PR DESCRIPTION
**Pull request by kshitij12345**
Ref #69991

### CODE REVIEW

File: aten/src/ATen/native/SpectralOps.cpp [hunk 1]
```diff
@@ -1100,8 +1100,11 @@ Tensor istft(const Tensor& self, const int64_t n_fft, const optional<int64_t> ho
...
-  const auto window_envelop_lowest = window_envelop.abs().min().item().toDouble();
-  if (window_envelop_lowest < 1e-11) {
+  const auto window_envelop_lowest = window_envelop.abs().min();
+  if (at::equal(
+          window_envelop_lowest.lt(1e-11),
+          window_envelop_lowest.new_ones(
+              {}, window_envelop_lowest.options().dtype(kBool)))) {
     std::ostringstream ss;
     REPR(ss) << "window overlap add min: " << window_envelop_lowest;
     AT_ERROR(ss.str());
```
Comment (kshitij12345) [inline aten/src/ATen/native/SpectralOps.cpp:1110]: Is the following code similar to using `TORCH_CHECK`? 
```c++
if (cond) {
   AT_ERROR(msg)
}
```

Comment (kshitij12345) [reply aten/src/ATen/native/SpectralOps.cpp:1107]: Eg.
https://github.com/pytorch/pytorch/blob/4a1783ee4c1d812953c2c059d2bafa384f4d0e92/aten/src/ATen/native/SpectralOps.cpp#L1104-L1108

**Comment by kshitij12345** [comment]
@zou3519 Gentle ping :)

**Review by zou3519 — APPROVED** [review]

File: aten/src/ATen/native/SpectralOps.cpp (hunk 1 above)
Comment (zou3519) [reply aten/src/ATen/native/SpectralOps.cpp:1107]: Yeah, this is how we used to do errors before the TORCH_CHECK macros were invented

**Pull request closed without merging**